package.json

```
@@ -85,10 +88,10 @@
     "services:stop": "npm run docker:compose -- stop",
     "services:down": "npm run docker:compose -- down",
     "docker:compose": "docker compose --env-file .env -f infra/docker-compose.development.yml",
-    "concurrently": "kill-port 3000 && npm run services:up && concurrently -s -k -n next,jest \"npm run next\"",
-    "test": "npm run concurrently -- \"jest --runInBand\" --hide next --verbose || npm run services:stop",
-    "test:watch": "jest --runInBand --watchAll",
-    "test:watch:services": "npm run concurrently -- \"jest --runInBand --watchAll\" || npm run services:stop",
+    "concurrently": "kill-port 3000 && npm run services:up && concurrently -s -k -n next,vitest \"npm run next\"",
+    "test": "npm run concurrently -- \"vitest run tests/unit\" --hide next --verbose || npm run services:stop",
```

Essa revisão me fez perceber que no #1621 eu acabei deixando o `--verbose` no lugar errado, fazendo os resultados dos testes serem apresentados apenas por arquivos ao invés de por teste individual.

Se acharem que compensa manter resumido assim, então é só remover o `--verbose` que não está fazendo nada aqui. O `tests/unit` eu imagino que foi algo esquecido durante a revisão.

Mas se acharem melhor voltar o relatório mais completo, então:

```suggestion
    "test": "npm run concurrently -- \"vitest run --reporter verbose\" --hide next || npm run services:stop",
```

---

> Se acharem que compensa manter resumido assim, então é só remover o `--verbose` que não está fazendo nada aqui.

Não utilizo os testes com tanta frequência então a remoção do `--verbose` não fez diferença pra mim, inclusive não tinha notado também que estava no lugar errado. Vou manter sem o `--verbose`, mas se mudarem de ideia posso voltar ele.

> O `tests/unit` eu imagino que foi algo esquecido durante a revisão.

Sim, tinha esquecido ele aí.

---

> Se acharem que compensa manter resumido assim, então é só remover o `--verbose` que não está fazendo nada aqui.

Precisei mexer bastante com testes no último PR que fiz e não percebi isso. Fui na [documentação](https://vitest.dev/guide/reporters) ver a diferença com/sem `--verbose` e acho que podemos continuar sem.